README.md

```diff
@@ -0,0 +1,36 @@
+# CoreLib: Astrolabe <--> Neo4j Integration
+
+## About 
+
+CoreLib is a Python module intended for integrating Neo4j with [Astrolabe](https://github.com/astrolabe-oss/astrolabe). 
+CoreLib is written using [neomodel](https://github.com/neo4j-contrib/neomodel)
+which is a Neo4j OGM that allows for OOP to be leveraged with the Neo4j database.
+
+CoreLib contains:
+* Mechanism for connection to Neo4j.
+* Classes used to represent various infrastructure components.
+    * All infrastructure classes are child classes of the PlatDB parent class.
+
+## Prerequisites 
+
+* \>=Python3.10
```

Comment:
Were there any other dependencies that I should mention here? Like neomodel? it's already in setup.py so I wasn't sure if that's just superfluous.

Reply:
`setup.py` should speak for itself and the README can/should only mention system level prerequisites such as python versions (although this can be codified as well) and any other system libraries needed. so this is fine!